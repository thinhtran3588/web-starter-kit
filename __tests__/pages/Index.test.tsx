import React from 'react';
import { render } from '@testing-library/react';
import Index from '../../pages';

describe('pages/Index', () => {
  beforeEach(() => {});

  it('renders successfully', async () => {
    const { baseElement } = render(<Index></Index>);
    expect(baseElement).toMatchSnapshot();
  });
});
