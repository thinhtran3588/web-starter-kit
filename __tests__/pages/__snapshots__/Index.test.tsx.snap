// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Index renders successfully 1`] = `
<body>
  <div>
    <div>
      <div
        class="jsx-1497064902"
      >
        <a
          class="jsx-1497064902"
          href="/"
        >
          Home
        </a>
        <a
          class="jsx-1497064902"
          href="/about"
        >
          About
        </a>
      </div>
      <h1>
        Home
      </h1>
    </div>
  </div>
</body>
`;
